arches_her/pkg/graphs/resource_models/Consultation.json

```diff
@@ -52,7 +52,7 @@
                         "overviewzoom": 11,
                         "pointradius": 4,
                         "selectRelatedSource": "select-application-area",
-                        "selectRelatedSourceLayer": "",
+                        "selectRelatedSourceLayer": "app-area",
```

Comment:
This should probably be done with the migration rather than changing the graph directly, otherwise upgrading instances will be difficult.

Reply:
Understood - I think @SDScandrettKint wasn't sure if the migration would work in the project instance he currently has ESCC's Arches for HERs instance running on

Reply:
@chiatt - Is this still something that needs to be applied by migration?

Reply:
@chiatt @csmith-he - as this is part of a graph JSON in the package for the 1.0.0 release, it doesn't need a migration. All future graph JSON changes after release 1.0.0 will need to be accompanied by a migration, which the arches app pattern will allow us to do.